packages/forms-web-app/src/nunjucks-configure.js

```diff
@@ -45,6 +46,7 @@ function nunjucksConfigure(app) {
 	nunjucksEnv.addFilter('formatMimeType', fileTypeDisplayHelper);
 	nunjucksEnv.addFilter('filterByKey', filterByKey);
 	nunjucksEnv.addFilter('render', renderTemplateFilter(nunjucks));
+	nunjucksEnv.addFilter('fileNameFromDocumentURL', getFileNameFromDocumentUrl);
```

Comment:
Is there an advantage to adding this functionality to nunjucks over determine the file name in the controller?

Reply:
this template is being used for several pages, (documents, representations, section-51) and It would need to include it in all those controllers. This touches less stuff and  it seemed like a handy functionality to have available